src/NuGet.Config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Many `git push -f` later and I still do not know what I've done to edit this line. :confounded: